src/warehouse/v1/util.js

```diff
@@ -120,6 +120,9 @@ function transformName(provider, name = '') {
   if (provider === 'postgres') {
     key = key.substr(0, 63);
   }
+  if (!options?.underscoreDivideNumbers) {
+    key = key.replace(/(\w)_(\d+)/g, '$1$2');
```

Comment:
Not sure if this is what we want to do:

It will work for `revisionV3` -> `revision_v3`. 

However, if the event input is `revision_3` we are going to rename it to `revision3`. Which doesn't seem correct to me.

Reply:
I would suggest using our own `_.snakeCase` implementation that doesn't split text from numbers in the first place.